lib/strategy/session.ex

```diff
@@ -9,9 +9,13 @@ defmodule Doorman.Strategy.Session do
     repo = Application.get_env(:doorman, :repo) # Constable.Repo
     user_module = Application.get_env(:doorman, :user_module) # Constable.User
 
-    case repo.get(user_module, id) do
-      nil -> {:error, "Could not find user"}
-      user -> {:ok, user}
+    if id == nil do
+      {:error, "Session's user_id is nil"}
```

Comment:
Instead of strings here, what do you think of making these module attributes?

eg: `@nil_user_id_error "Session's user_id can't be nil"`

I think you could then you could use `Module.get_attribute(Doorman.Strategy.Session, @nil_user_id_error)` in your test instead of the raw string.